plugin.program.chrome.launcher/resources/language/English/strings.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<strings>
+  <string id="30001">Add Website</string>
```

Comment:
In your next update, please upgrade to strings.po, rather than strings.xml which is deprecated.